htmd/parameterization/test.py

```diff
@@ -7,235 +7,200 @@
 import sys
 import shutil
 import unittest
-from tempfile import TemporaryDirectory
 from subprocess import call
+from htmd.home import home
+from htmd.util import tempname
+
+os.environ['HTMD_LONGTESTS'] = 'yes'
 
 
 class TestParameterize(unittest.TestCase):
 
     def setUp(self):
 
         self.maxDiff = None
-
-        self.dataDir = os.path.join('..', 'data', 'test-param')
-        if not os.path.exists(self.dataDir):
-            self.dataDir = os.path.join('htmd', 'data', 'test-param')
-
-        suffix = 'travis' if 'TRAVIS' in os.environ else 'local'
-        self.dataDir = os.path.abspath(os.path.join(self.dataDir, suffix))
+        self.dataDir = home(dataDir='test-param')
 
         if os.environ.get('TRAVIS_OS_NAME') == 'osx':
             self.skipTest('Mac does not work!')
 
-    def _test(self, refDir, resDir):
+    def _test(self, refDir, resDir, cmd):
+
+        if not os.path.exists(resDir):
+            os.makedirs(resDir)
 
-        shutil.copy(os.path.join(refDir, 'stdin'), resDir)
         molFile = os.path.join(refDir, 'input.mol2')
         if os.path.exists(molFile):
             shutil.copy(molFile, resDir)
 
-        with open(os.path.join(resDir, 'stdin')) as stdin, \
-             open(os.path.join(resDir, 'stdout'), 'w') as stdout, \
-             open(os.path.join(resDir, 'stderr'), 'w') as stderr:
-            arguments = stdin.readline().split()
-            returncode = call(arguments, stdout=stdout, stderr=stderr, cwd=resDir)
-            self.assertEqual(returncode, 0)
-
-        paths = [[os.path.join(directory, file) for file in files] for  directory, _, files in os.walk(refDir)]
-        paths = sum(paths, [])
-        paths = [os.path.relpath(path, refDir) for path in paths]
-
-        paths = [path for path in paths if not path.startswith('minimize')]
-        paths = [path for path in paths if not path.startswith('esp')]
-        paths = [path for path in paths if not path.startswith('dihedral')]
-        paths = [path for path in paths if not path.endswith('.coor')]
-        paths = [path for path in paths if not path.endswith('.svg')]
-
-        self.assertTrue('stdout' in paths)
-        #self.assertTrue('stderr' in paths)
-
-        for path in paths:
-            refFile, resFile = os.path.join(refDir, path), os.path.join(resDir, path)
-
-            with self.subTest(path=path):
-                self.assertTrue(os.path.exists(refFile))
+        arguments = cmd.split()
+        returncode = call(arguments, cwd=resDir)
```

Comment:
Even if we don't test logs, it would be useful to see stdout and stderr to understand crashes, etc.